app/activity_notifications/request_scheduled.rb

```diff
@@ -28,6 +28,7 @@ def group_message(notifications:)
       request_title: request.title,
       date: I18n.l(request.schedule_send_for, format: '%A, %-d.%-m.%Y'),
       time: I18n.l(request.schedule_send_for, format: '%H:%M'),
+      # probably scope by request.organization?
```

Comment:
I don't think so as the activity notifications are scoped by the organization so only requests from that org appear in the dashboard. This is only accessing the time the request is scheduled for and displaying it.

We need to be careful, we don't have activity notifications of type request scheduled that have requests from other organizations, but other than that, at this level there is no need for additional scoping.

We create this type of notification in an after create commit here https://github.com/tactilenews/100eyes/blob/main/app/models/request.rb#L74 and we add the the organization id:

```rb
RequestScheduled.with(request_id: request.id, organization_id: request.organization.id).deliver_later(User.all)
```

Then in the dashboard controller, we only look for the notifications that are for that current user scoped by the organization id we have set. See https://github.com/tactilenews/100eyes/blob/main/app/controllers/dashboard_controller.rb#L16

Note that the :point_up: request scheduled would cause leaky notifications. See https://github.com/tactilenews/100eyes/pull/1983